cmd/init.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"os"
+
+	templatesBox "github.com/okkur/reposeed/cmd/templates"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(initCmd)
+}
+
+var initCmd = &cobra.Command{
+	Use:     "init",
+	Short:   "Create the sample config file",
+	Example: "reposeed init ~/myproject",
+	Run:     initHandler,
+}
+
+func initHandler(cmd *cobra.Command, args []string) {
+	box := templatesBox.GetTemplates()
+	seedString := box.String("seed-config.example.yaml")
+	file, _ := os.Create(args[0] + "/.seed-config.yaml")
```

Comment:
On an empty args[0] wouldn't this lead to creating the file at the root of the file system?
Aka "reposeed init" would not create it in the current directory.
What's the reasoning not to use --output here as well + a default to .seed-config.yaml?
+ a warning, if the file already exists and the ability to overwrite using the global overwrite flag?

Reply:
I agree that using the flags makes more sense and even makes the code easier to understand. Applied the requested changes in the last commit.